spec/helpers/comments_helper_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe CommentsHelper do
+  describe "#comment_path" do
+    let(:comment) { VCR.use_cassette('planningalerts') { create(:confirmed_comment) } }
+
+    it "returns the path for the application with an anchor for the comment" do
+      expect(helper.comment_path(comment)).to eq application_path(comment.application, anchor: "comment#{comment.id}")
```

Comment:
I see what you mean. Something could happen that caused `comment.id` to return the wrong thing and this would still pass.

I'll update the test cause I think it’s better to keep the coverage so any changes to the helper are very intentional.